hack/install-konveyor.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Figure out where we are being run from.
+# This relies on script being run from:
+#  - ${PROJECT_ROOT}/hack/install-tackle.sh
+#  - ${PROJECT_ROOT}/bin/install-tackle.sh
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+__root="$(cd "$(dirname "${__dir}")" && pwd)"
+__repo="$(basename "${__root}")"
+__bin_dir="${__root}/bin"
+__os="$(uname -s | tr '[:upper:]' '[:lower:]')"
+__arch="$(uname -m | sed 's/x86_64/amd64/' | sed 's/aarch64/arm64/')"
+
+# Update PATH for execution of this script
+export PATH="${__bin_dir}:${PATH}"
```

Comment:
$PATH is always exported, no need, to re-export it

Reply:
I didn't realize this. Will examine.